jupyter_alabaster_theme/static/pcss/mobile_sidebar.pcss

```diff
@@ -1,72 +1,85 @@
-/* Mobile Navbar Styling */
-.mobile-nav-current {
-  max-width: 90%;
-  display: inline-block;
-}
+/* Mobile NavBar Styling */
+.mobile-nav- {
+  &current {
+    max-width: 90%;
+    display: inline-block;
 
-.mobile-nav-current-dropdown {
-    padding: 16px 20px;
-    border: 2px solid #BDBDBD;
-    border-radius: 4px;
-    margin-bottom: 12px;
-}
+    &-dropdown {
```

Comment:
Wow, surprised this works in the middle of a sector. But looks like the CSS is what you would expect. Very cool!